### PR TITLE
[argo/k8s] add terminationMessagePolicy 

### PR DESCRIPTION
**Pull request by iamsgarg-ob**
- add a terminationMessagePolicy to FallbackToLogsOnError

### CODE REVIEW

File: metaflow/plugins/argo/argo_workflows.py [hunk 1]
```diff
@@ -2015,6 +2015,7 @@ def _container_templates(self):
                         kubernetes_sdk.V1Container(
                             name=self._sanitize(node.name),
                             command=cmds,
+                            termination_message_policy= "FallbackToLogsOnError",
```
Comment (savingoyal) [inline metaflow/plugins/argo/argo_workflows.py:2018]: can you also enable it for `@kubernetes` so that there is parity?

Comment (iamsgarg-ob) [reply metaflow/plugins/argo/argo_workflows.py]: done.

File: metaflow/plugins/kubernetes/kubernetes_jobsets.py
Comment (valayDave) [inline metaflow/plugins/kubernetes/kubernetes_jobsets.py:1]: Can you remove todo on L760 ?

File: metaflow/plugins/kubernetes/kubernetes_job.py
Comment (valayDave) [inline metaflow/plugins/kubernetes/kubernetes_job.py:1]: Can you remove todo in L260?

**Review by savingoyal — APPROVED** [review]

**Review by valayDave — APPROVED** [review]
LGTM, outside the small changes to remove todos about `termination_message_policy` in the code.

File: metaflow/plugins/kubernetes/kubernetes_job.py
Comment (iamsgarg-ob) [reply metaflow/plugins/kubernetes/kubernetes_job.py:1]: https://github.com/Netflix/metaflow/pull/1935/files#diff-7071166b2208ea44f84aa14674edd8449a01d3708bd0e0e107d74aecf43f449aL260

File: metaflow/plugins/kubernetes/kubernetes_jobsets.py
Comment (iamsgarg-ob) [reply metaflow/plugins/kubernetes/kubernetes_jobsets.py:1]: https://github.com/Netflix/metaflow/pull/1935/files#diff-333561ac259e0c08a4933e9da3826bf949127132bb1682519129b51d9ed9f1e4L760

**Pull request merged**